runtime/crab/polkadot-compatible-types.json

```diff
@@ -243,6 +243,7 @@
 			"_enum": {
 				"Any": null,
 				"NonTransfer": null,
+				"ProxyType": null,
```

Comment:
A bit confused:

What are `polkadot-compatible-types.json` and `type.json` used for?  They have extractly the same content.

Reply:
No, their content are different

Reply:
One for darwinia apps another for polkadot apps